ui-library-filesharing-chat-composite/README.md

```diff
@@ -0,0 +1,68 @@
+# Azure Communication Services application with file sharing
+
+This is a complete sample showcasing integration of file sharing feature
+into an [Azure Communication Services][docs-root] powered [React] application.
+
+For a step-by-step guide to building this sample yourself, see the associated tutorial at MSDN.
+
+    TODO: Link to tutorial once available.
+
+This sample includes:
+* A React frontend built using the UI chat composite provided by the [Azure Communication Services UI library][docs-ui-library].
+* Backend API for call recording integration built using [Typescript Azure Functions][typesctipt-azure-functions].
+
+
+[docs-root]: https://docs.microsoft.com/en-us/azure/communication-services/
+[docs-ui-library]: https://azure.github.io/communication-ui-library/
+[typesctipt-azure-functions]: https://docs.microsoft.com/en-us/azure/azure-functions/create-first-function-vs-code-typescript
+[React]: https://reactjs.org/
+
+
+## Code organization
+
+* [app/](./app) - This directory contains source code for the React frontend.
+  * [app/src/App.tsx](./app/src/App.tsx) - Application entry-point. Built primarily using UI library's [ChatComposite](https://azure.github.io/communication-ui-library/?path=/docs/composites-chat-basicexample--basic-example).
+  * [app/src/Api.ts](./app/src/Api.ts) - client side bindings to interact with the backend API from [api/](./api).
```

Comment:
There's no Api.ts file in this project.
Let's double check the text added to this project README files to make sure it is reflective of the contents in this project.